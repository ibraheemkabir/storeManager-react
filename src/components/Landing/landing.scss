.home {
  height: 100%;
  background-color: #2550b1eb;

  span {
    white-space: normal;
    overflow: hidden;
    padding: 2rem;
    width: 55%;
    text-shadow: 2px 2px 2px #000;

    p {
      text-shadow: none;
      a {
        color: white;
      }
      button.ui.google.plus.button {
        height: 3rem;
        width: 40%;
      }
      .ui.horizontal.divider {
        width: 30%;
        margin: 0.5rem auto;
      }
    }
  }
}

.maintext {
  width: 100%;
  height: 100vh;
  color: white;
  font-size: 4rem;
  line-height: 1.2;
  text-align: center;
  vertical-align: middle;
  display: flex;
  align-items: center;
  font-weight: 600;
}

.logo {
  animation: typing 1s steps(10, end), blink 1s step-end infinite;
  border-right: 1px solid orange;
  display: inline-block;
  overflow: hidden;
  position: absolute;
  padding: 1rem;
  font-size: 2rem;
  color: white;
  font-family: monospace;
  font-weight: bolder;
  margin: 0.2rem;
  letter-spacing: 0.1rem;
}

@keyframes typing {
  from {
    width: 0%;
  }
  to {
    width: 10%;
  }
}

@keyframes blink {
  from,
  to {
    border-color: transparent;
  }
  50% {
    border-color: orange;
  }
}

.hero {
  width: 45%;
  img {
    border-style: none;
    margin-left: -10rem;
  }
}
