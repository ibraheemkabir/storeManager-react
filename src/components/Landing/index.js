import './landing.scss';
import React, { Component } from 'react';
import { Button, Icon, Divider } from 'semantic-ui-react';
import {Link} from 'react-router-dom';

export default class index extends Component {
  render() {
    return (
      <div>
        <div className="logo">STORELY</div>
        <div className="maintext">
          <span>
            {' '}
            STORE MANAGEMENT WITH EASE AND CONVINIENCE.
            <p>
              <Button color="google plus">
                <Icon name="google plus" />
                {' '}
Sign in with Gmail
              </Button>
              <Divider horizontal>
                <Link to='/login'>Or Log In</Link>
              </Divider>
            </p>
          </span>
          <div className="hero">
            <img src="https://res.cloudinary.com/ah-krypton/image/upload/c_scale,w_800/v1552771277/shop.png" />
          </div>
        </div>
      </div>
    );
  }
}
