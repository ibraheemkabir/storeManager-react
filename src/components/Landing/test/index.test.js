import React from 'react'
import {shallow} from 'enzyme';
import Landing from '../index';
import { Item } from 'semantic-ui-react';

describe('test for landing page',()=>{
    it('should test for landing page component',()=>{
        const wrapper = shallow(<Landing/>)
        expect(wrapper).toMatchSnapshot();

    })
})