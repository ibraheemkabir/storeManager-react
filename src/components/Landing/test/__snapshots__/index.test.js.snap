// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for landing page should test for landing page component 1`] = `
<div>
  <div
    className="logo"
  >
    STORELY
  </div>
  <div
    className="maintext"
  >
    <span>
       
      STORE MANAGEMENT WITH EASE AND CONVINIENCE.
      <p>
        <Button
          as="button"
          color="google plus"
        >
          <Icon
            as="i"
            name="google plus"
          />
           
          Sign in with Gmail
        </Button>
        <Divider
          horizontal={true}
        >
          <Link
            replace={false}
            to="/login"
          >
            Or Log In
          </Link>
        </Divider>
      </p>
    </span>
    <div
      className="hero"
    >
      <img
        src="https://res.cloudinary.com/ah-krypton/image/upload/c_scale,w_800/v1552771277/shop.png"
      />
    </div>
  </div>
</div>
`;
